ovs/lib/vdisk.py

```diff
@@ -581,6 +584,37 @@ def delete_snapshots(snapshot_mapping):
         :return: Information about the deleted snapshots, whether they succeeded or not
         :rtype: dict
         """
+
+        def _delete_multiple_snapshots(vdisk, snapshot_ids):
+            # type: (ovs.dal.hybrids.vdisk, List[str]) -> Dict[str, Dict[str, Dict[str, List[str, str]]]]
+            if getattr(vdisk.storagedriver_client, 'delete_snapshots', None):
+                results[vdisk.guid]['results'] = VDiskController.delete_snapshots_concurrent(vdisk, snapshots=snapshot_ids)
+            else:
+                for snapshot_id in set(snapshot_ids):
```

Comment:
Make it a separate function and return the same format as `VDiskController.delete_snapshots_concurrent`